### ISSUE TITLE
Command is executed async

### ISSUE THREAD

**Issue by MWFIAE** [Co0sh/BetonQuestUploader#4]
[13:30:24 INFO]: [BetonQuestUploader] mwfiae uploaded his package.
[13:30:24 ERROR]: Command Dispatched Async: betonquest reload
[13:30:24 ERROR]: Please notify author of plugin causing this execution to fix this bug! see: http://bit.ly/1oSiM6C
java.lang.Throwable: null
        at org.bukkit.craftbukkit.v1_12_R1.CraftServer.dispatchCommand(CraftServer.java:658) ~[PaperSpigot-1.12.1-b1204.jar:git-Paper-1204]
        at org.bukkit.Bukkit.dispatchCommand(Bukkit.java:590) ~[PaperSpigot-1.12.1-b1204.jar:git-Paper-1204]
        at pl.betoncraft.betonquestuploader.Receiver.receive(Receiver.java:289) ~[?:?]
        at pl.betoncraft.betonquestuploader.Receiver.lambda$onEnable$0(Receiver.java:73) ~[?:?]
        at com.sun.net.httpserver.Filter$Chain.doFilter(Filter.java:79) [?:1.8.0_144]
        at sun.net.httpserver.AuthFilter.doFilter(AuthFilter.java:83) [?:1.8.0_144]
        at com.sun.net.httpserver.Filter$Chain.doFilter(Filter.java:82) [?:1.8.0_144]
        at sun.net.httpserver.ServerImpl$Exchange$LinkHandler.handle(ServerImpl.java:675) [?:1.8.0_144]
        at com.sun.net.httpserver.Filter$Chain.doFilter(Filter.java:79) [?:1.8.0_144]
        at sun.net.httpserver.ServerImpl$Exchange.run(ServerImpl.java:647) [?:1.8.0_144]
        at sun.net.httpserver.ServerImpl$DefaultExecutor.execute(ServerImpl.java:158) [?:1.8.0_144]
        at sun.net.httpserver.ServerImpl$Dispatcher.handle(ServerImpl.java:431) [?:1.8.0_144]
        at sun.net.httpserver.ServerImpl$Dispatcher.run(ServerImpl.java:396) [?:1.8.0_144]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_144]

Basically, start a new Sync-Thread in the uploader plugin, so you don't have to worry inside the betonquest command.
Thats all :)
